src/containers/WelcomePage/FrontpageSubjects.jsx

```diff
@@ -137,51 +152,68 @@ const LinkToAbout = () => (
   </a>
 );
 
-class FrontpageSubjects extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { preview: false };
-  }
-
-  componentDidMount() {
+const FrontpageSubjects = ({ categories, subjects, locale }) => {
+  const [preview, setPreview] = useState(false);
+  const [showImported, setShowImported] = useState(false);
+  useEffect(() => {
     if (
       document.location.search &&
```

Comment:
sjekka at dette fortsatt fungerer med SSR? Kan testes ved å ctrl + r på selve sida.

Reply:
it works